### PR TITLE
Actuals and tier1

### PR DESCRIPTION
**Pull request by jmccrosky**

### CODE REVIEW

File: simpleprophet/simpleprophet/output.py [hunk 1]
```diff
@@ -63,6 +62,23 @@ def prepare_records(modelDate, forecast_end, data, product):
     models = setup_models(years)
     forecast_start = modelDate + timedelta(days=1)
     forecast_period = pd.DataFrame({'ds': pd.date_range(forecast_start, forecast_end)})
+    actuals_data = {
+        "asofdate": modelDate,
+        "datasource": product,
+        "date": data.ds,
```
Comment (jklukas) [inline simpleprophet/simpleprophet/output.py:69]: Does our code ensure that `data` includes dates only less than `asofdate`? We should make sure to always have a continuous range of dates where the row with `asofdate = date` is the last day with `type = 'actual'` and the first date with type `forecast` is `asofdate + 1`.

**Review by jklukas — APPROVED** [review]
This looks great other than making sure we have no overlap between forecast and actuals. We probably need to make a change in the queries to ensure that or we can truncate `data` within `prepare_records`.

File: simpleprophet/simpleprophet/output.py (hunk 1 above)
Comment (jmccrosky) [reply simpleprophet/simpleprophet/output.py:69]: I think I fixed this.  You want to confirm it looks OK?

Comment (jklukas) [reply simpleprophet/simpleprophet/output.py:69]: Looks good.

**Pull request merged**